ci/environment-dev.yaml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - Cython>=0.28.2
   - NumPy
   - flake8
+  - flake8-comprehensions
```

Comment:
I think need to run and commit scripts/convert_deps.py to have requirements_dev.txt updated. can you PR that?